app/views/return-requirements/points.njk

```diff
@@ -15,14 +18,51 @@
 {% endblock %}
 
 {% block content %}
+  {# Error summary #}
+  {% if error %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: [
+        {
+          text: "Select any points for the return requirement",
+          href: "#points"
+        }
+      ]
+    }) }}
+  {%endif%}
+
   {# Main heading #}
-  <div class="govuk-body">
+  <div>
+    <span class="govuk-caption-l"> Licence {{ licenceRef }} </span>
     <h1 class="govuk-heading-xl govuk-!-margin-bottom-3">{{ pageTitle }}</h1>
+
+
+    <div class="govuk-hint"> Select all that apply </div>
   </div>
 
+<div>
   <form method="post">
-    <div class="govuk-body">
-      {{ govukButton({ text: "Continue" }) }}
-    </div>
+      {% set checkBoxItems = [] %}
+      {% for point in licencePoints %}
+
+        {% set checkBoxItem = {
+          value: point,
+          text: point,
+          checked: false
+        } %}
+
+      {% set checkBoxItems = (checkBoxItems.push(checkBoxItem), checkBoxItems) %}
+
+      {% endfor %}
+        {{ govukCheckboxes({
+          name: "points",
+          errorMessage: error,
+          items: checkBoxItems
+        }) }}
+
+    {{ govukButton({ text: "Continue" }) }}
   </form>
+</div>
```

Comment:
It feels like the alignment has got a little out of whack here. The `<div>` isn't inline with the section above and the elements inside the form appear to be double indented.

I'd suggest giving it another look.